backend/long_running_request_actors.py

```diff
@@ -13,28 +13,34 @@
 PRIORITY_HIGH = 0
 
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "backend.settings")
```

Comment:
Please put this back. It's very very important that the code actually runs. 

If you dont want this to run during unit testing then just use an if statement.

If you dont know what a piece of code is for then it's not good to just delete it. This would break prod completely, none of our dramatiq stuff would work without it because the worker wouldn't be able to access the db.

Reply:
I am not sure why it got deleted, also, the print statement and the import I did not put there.  I apologise for this, it is definitely not something I deliberately did.